content/millennium/r4/scheduling/appointment.md

```diff
@@ -32,6 +32,7 @@ The following fields are returned if valued:
 
 * [Appointment id](http://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
 * [Status](http://hl7.org/fhir/R4/appointment-definitions.html#Appointment.status){:target="_blank"}
+* [Cancellation Reason](http://hl7.org/fhir/R4/appointment-definitions.html#Appointment.cancelationReason){:target="_blank"}
```

Comment:
Do we not need a terminology binding for this?

Reply:
I am not sure about the terminology binding. I would need some opinion on this maybe.
@exopsRamba what do you think?

Reply:
I think we need terminology binding as we are mapping millennium codeset to standard code system

Reply:
Got it. Terminology bindings added, and snapshots added.
<img width="999" alt="Screenshot 2021-03-18 at 5 54 31 PM" src="https://user-images.githubusercontent.com/56925992/111626602-3aabe780-8814-11eb-80d4-75dd3046e7e2.png">
<img width="975" alt="Screenshot 2021-03-18 at 5 54 51 PM" src="https://user-images.githubusercontent.com/56925992/111626622-43042280-8814-11eb-92a0-be0d5eb18d2e.png">
<img width="923" alt="Screenshot 2021-03-18 at 5 55 14 PM" src="https://user-images.githubusercontent.com/56925992/111626631-44cde600-8814-11eb-889c-460f5d7f0476.png">


Changes:
https://github.com/cerner/fhir.cerner.com/commit/c44de0c25175498659d8a60f64949a635c699b17#diff-aedba7f37b773d7a314400db492774b8a9d406ed43aad2101793b82970106103R205

https://github.com/cerner/fhir.cerner.com/commit/c44de0c25175498659d8a60f64949a635c699b17#diff-aedba7f37b773d7a314400db492774b8a9d406ed43aad2101793b82970106103R607

https://github.com/cerner/fhir.cerner.com/commit/c44de0c25175498659d8a60f64949a635c699b17#diff-256db1c27cbe3136db6001a65a04bd3cffaf5beb61789a01f2c6da8b2676bc73R28

Reply:
Do we need the fhir value set listed as well? Appointment cancellation reason -  https://www.hl7.org/fhir/r4/valueset-appointment-cancellation-reason.html

Reply:
Done,
https://github.com/cerner/fhir.cerner.com/commit/c28a4715d5e50613d25066910c37d68feca097bd#diff-256db1c27cbe3136db6001a65a04bd3cffaf5beb61789a01f2c6da8b2676bc73R51

Reply:
Yeah, i believe it should match the system we are returning to match other resources: http://terminology.hl7.org/CodeSystem/appointment-cancellation-reason

Reply:
Added
https://github.com/cerner/fhir.cerner.com/commit/c28a4715d5e50613d25066910c37d68feca097bd#diff-256db1c27cbe3136db6001a65a04bd3cffaf5beb61789a01f2c6da8b2676bc73R51

changed the system again.
https://github.com/cerner/fhir.cerner.com/commit/b55208c1d4716f3bb4b84ae86fe301aefaf1016e#diff-256db1c27cbe3136db6001a65a04bd3cffaf5beb61789a01f2c6da8b2676bc73R52

Reply:
<img width="983" alt="Screenshot 2021-03-18 at 8 01 06 PM" src="https://user-images.githubusercontent.com/56925992/111643132-b6ae2b80-8824-11eb-9b09-bbbab2d7689b.png">

Reply:
+1